prems-resources/clinical-data/src/main/resources/SLING-INF/content/apps/cards/clinics/UHN-IC-ED/mailTemplates/InitialInvitation/bodyTemplate.html

```diff
@@ -0,0 +1,49 @@
+    <p>Dear ${first_name} ${last_name},</p>
+    <p>
+      You are receiving this email because of your recent visit at University Health Network (UHN) hospital within the last 30 days.
+      We would like you to reflect on your experience with the Integrated Care Program as you transitioned home and to provide us
+      with some feedback on supports that you may have received.
+    </p>
+    <p>
+      Your opinions are valuable to us.
+      Your responses will be anonymous and will be used to help UHN continuously improve the quality of care patients receive. 
+    </p>
+    <p class="surveylink">
+      <a href="${surveysLink}">Complete the survey</a>
+    </p>
+    <p>
+      Completing this survey is voluntary. Your responses will not be shared with the care
+      team and answering will not impact future care you receive at UHN.
+    </p>
+    <p>
+      The questionnaire contains 7 questions and will take about 5 minutes to complete.
+    </p>
+    <p>
+      <strong>The survey will be available until ${expirationDate}. Please do not delay.</strong>
+    </p>
+
+    <div class="note">
+      <p>
+        If the button does not work for you, please copy this web address and paste it in your web browser's address bar:
+        <br/>
+        <span class="web-address">${surveysLink}</span>
+      </p>
+      <hr/>
+      <p>
+        The survey will not load if you are using Internet Explorer. Please use another web browser, such as Edge, Chrome, or Firefox.
+      </p>
+    </div>
+    <p>
+      If you have any questions, concerns, or wish to be removed from the survey, please contact us at yourexperience@uhn.ca.
```

Comment:
For all emails inviting patients to fill out an IC survey (even if bundled with other surveys), the contact email address and the reply-to address should be `Ic_Patientexperience@uhn.ca`.

```suggestion
      If you have any questions, concerns, or wish to be removed from the survey, please contact us at Ic_Patientexperience@uhn.ca.
```